cockatrice/src/settings/shortcuts_settings.h

```diff
@@ -536,7 +539,7 @@ class ShortcutsSettings : public QObject
                                                 parseSequenceString(""),
                                                 ShortcutGroup::Move_bottom)},
         {"Player/aDrawArrow", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Draw Arrow..."),
-                                          parseSequenceString("Ctrl+A"),
+                                          parseSequenceString("Alt+A"),
```

Comment:
I'm okay with this change

Reply:
people can learn a new shortcut but I think if someone has set this shortcut and we overwrite it some unexpected things might happen, this'd need some testing